python/constants.py

```diff
@@ -21,3 +21,5 @@ class PyRasgoEnvironment(Enum):
 # TODO: When we support multiple DWs in the API
 # we'll need to refactor the functions that consume this
 RASGO_DATAWAREHOUSE = 'snowflake'
+
+COMMUNITY_ORGANIZATION_ID: int = 1
```

Comment:
I'm loving that this is going in as a var, but I wonder if it should be an enum; it's possible that we have diff org IDs we want to publish with by environment (if not now, potentially in the future). Not sure that it blocks right now, tho

Reply:
May not to be bad to add if we ever need it, but as this is just our utils for now, I'm not too worried about perfect structure / plan for future as we might nix this all in a few months